tasks.py

```diff
@@ -153,6 +153,8 @@ def lint(
         cmds.append("--unsafe-fixes")
     if watch:
         cmds.append("--watch")
+    if os.getenv("GITHUB_ACTIONS"):
```

Comment:
Is this always set in github? It allows integration with their UI?

Reply:
Yes exactly.
https://docs.astral.sh/ruff/integrations/#github-actions